NuRadioReco/framework/electric_field.py

```diff
@@ -127,6 +129,18 @@ def set_position(self, position):
         """
         self._position = position
 
+    def set_position_onsky(self, position):
+        """
+        set position of the electric field in the on sky (vxB, vxvxB) coordinate system
+        """
+        self._position_onsky = position
+
+    def get_position_onsky(self):
+        """
+        get position of the electric field in the on sky (vxB, vxvxB) coordinate system
+        """
+        return self._position_onsky
+
```

Comment:
Why do we need to store these coordinates? Aren't they just a transformation of `position`?

Reply:
I am actually also confused by the docstring. The on-sky CS is not (vxB, vxvxB) right? It should be the (eTheta, ePhi, eR)?

Reply:
But I stand by my previous comment. Please do not store two potentially different positions!

Reply:
so  (vxB, vxvxB) is definitely wrong. But why can't we transform them once and store them?